src/main/webapp/pages/templates/galaxy_auth_code.tmpl.html

```diff
@@ -1,28 +1,54 @@
-<!doctype html>
-<html xmlns:th="http://www.thymeleaf.org"
-      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
-<head>
-	<title data-layout-title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE" th:text="#{global.title}">IRIDA Platform</title>
-</head>
-<body vocab="http://schema.org/" typeof="WebPage" ng-app="GalaxyRedirect" ng-controller="authCodeController">
-
-<th:block layout:fragment="pageScripts">
-	<script th:src="@{/resources/bower_components/angular/angular.min.js}"
-	        src="/resources/bower_components/angular/angular.min.js"></script>
-	<script th:inline="javascript">
-		angular.module('GalaxyRedirect', [])
-			.controller('authCodeController', ['$window', authCodeController]);
-
-		function authCodeController($window) {
-
-			getParentScope().$emit('galaxyAuthCode', /*[[${auth_code}]]*/ '');
-
-			function getParentScope() {
-				return $window.parent.angular.element($window.frameElement).scope();
-			}
-		}
-	</script>
-</th:block>
-
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <link
+      rel="stylesheet"
+      th:href="@{/resources/dist/css/access_confirmation.bundle.css}"
+    />
+    <style>
+      html,
+      body,
+      .wrapper {
+        height: 100%;
+      }
+      .wrapper {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+      }
+      p {
+        text-align: center;
+        font-weight: 600;
+        padding-top: 1rem;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="wrapper">
+      <i aria-label="icon: check-circle" class="anticon anticon-check-circle">
+        <svg
```

Comment:
Should SVG objects be moved to a separate file instead of being encoded directly in the HTML document?

Reply:
SVG is a valid html tag and is perfectly fine here.  It is just the checkmark.

Reply:
I guess I mean, is it going to be re-used in any other place in the code, or is this the only location it is going to be used?

Reply:
I don't think it is worth extracting at this point, this is pretty unique (so far)

Reply:
Okay, sounds good.